.circleci/config.yml

```diff
@@ -106,6 +106,9 @@ jobs:
       - run:
           name: Install dependencies
           command: python -m scripts.install_third_party_libs
+      - run:
+          name: Run test coverage
```

Comment:
This is too vague. Please change to "Checking that all e2e test files are captured in protractor.conf.js."

Reply:
(Sorry, missed this before. Found it when I was checking on the new tests to see if they were running correctly.)

Reply:
no worries, done! I worded it in present tense to keep it consistent with the other descriptions.